docs/docs/aztec/concepts/pxe/index.md

```diff
@@ -8,44 +8,101 @@ importance: 1
 
-- **Notes**: Encrypted representations of assets.
-- **Deferred Notes**: Notes that are intended for a user but cannot yet be decoded due to the associated contract not being present in the database. When new contracts are deployed, there may be some time before it is accessible from the PXE database. When the PXE database is updated, deferred note are decoded.
-- **Authentication Witnesses**: Data used to approve others from executing transactions on your behalf.
+- **Notes**: Data representing users' private state. These are often stored on-chain, encrypted to a user. The PXE will parse on-chain data to find notes relevant for accounts that it is aware of.
+- **Deferred Notes**: Notes that are intended for a user but cannot yet be decoded due to the associated contract not being present in the database. When new contracts are deployed, there may be some time before it is accessible from the PXE database. When the PXE database is updated, deferred notes are decoded.
+- **Authentication Witnesses**: Data used to approve others for executing transactions on your behalf. The PXE provides this data to transactions on-demand during transaction simulation via oracles.
 - **Capsules**: External data or data injected into the system via [oracles](#oracles).
+- **Address Book**: A list of expected addresses that a PXE may encrypt notes for, or received encrypted notes from. This list helps the PXE reduce the amount of work required to find notes relevant to it's registered accounts.
 
 ### Note discovery
 
-There is an open RFP for how note discovery will work on Aztec. You can find more information in the [forum](https://forum.aztec.network/t/request-for-proposals-note-discovery-protocol/2584).
+Note discovery helps solve the problem of a user parsing the chain for their encrypted notes.
+
+There is a note tagging scheme that allows users to register an expected note sender in their PXE, which tells the PXE to expect notes coming from that expected sender. This helps the PXE more efficiently discover the encrypted notes being created for it's registered accounts.
+
+If communication between transacting parties is not possible before transacting, users can use a brute force method of trial decrypting all created notes in order to determine which notes are intended for them.
+
+### Authorization
+
+The PXE handles access rights by:
+
+1. action
+2. domain
+3. contract
+4. account
+
+For example, uniswap.com (**domain**) can query (**action**, involves execution that has access to an accounts' private state) on these five token **contracts** for these two **accounts** of mine, that are registered in the PXE.
+
+Available actions include:
+
+- Seeing that the accounts exist in the PXE
+- Running queries, simulations, accessing logs, registering contracts, etc at a given a contract address
+- Manually adding notes
+
+Providing an application with no scopes to the PXE means that no information can be accessed.
```

Comment:
Careful with the terms here. An empty scopes array (i.e. `scopes: []`) means no information can be accessed, but _no_ scopes (i.e. `scopes: undefined`) defaults instead to _all_ scopes being available (it is the shortcut to avoid having to think about them).

We'll want to change this in the future because it is a dangerous default, but that's the current state of affairs.

Reply:
noted and clarified